packages/table/src/regions.ts

```diff
@@ -548,6 +544,42 @@ export class Regions {
         }
     }
 
+    private static maybeFullyOverlapsRegion(regions: IRegion[],
```

Comment:
`maybe` typically denotes _behavior_ that may or may not execute. `maybeRender`, `maybeHandle`.

this is an `is` situation since it returns a boolean.